ql/src/test/org/apache/hadoop/hive/ql/TxnCommandsBaseForTests.java

```diff
@@ -241,17 +242,20 @@ public static String makeValuesClause(int[][] rows) {
   }
 
   public static void runInitiator(HiveConf hiveConf) throws Exception {
-    runCompactorThread(hiveConf, CompactorThreadType.INITIATOR);
+    runCompactorThread(hiveConf, CompactorThreadType.INITIATOR, Constants.COMPACTION_DEFAULT_POOL);
```

Comment:
no need for changes here, just keep the original signature and use defauls
````
private static void runCompactorThread(HiveConf hiveConf, CompactorThreadType type) {
    runCompactorThread(hiveConf, type, Constants.COMPACTION_DEFAULT_POOL)
}
````

Reply:
Done